src/components/RestCourses/EditCourseForm.js

```diff
@@ -112,7 +112,10 @@ function EditCourseForm(props) {
         <Modal
           visible={props.visible}
           onCancel={props.hideModal}
-          onOk={submitForm}
+          onOk={() => {
+            submitForm();
```

Comment:
Since we're not using the event, can you please remove the unused `e` param from line 37, please?